src/icons.codegen.js

```diff
@@ -0,0 +1,175 @@
+const revFile = require('rev-file');
+const glob = require('glob');
+const path = require('path');
+
+const icons = glob
+  .sync('images/*icons.svg', {
+    cwd: path.join(__dirname),
+  })
+  .reduce((acc, next) => {
+    const key = path.basename(next).split('.')[0];
+
+    acc[key] = path.relative(
+      __dirname,
+      revFile.sync(path.join(__dirname, next))
+    );
+    return acc;
+  }, {});
+
+// subjectIcons.forEach(path => {
```

Comment:
why commented?

Reply:
Ah it was just temp code. I will remove it :)